hack/release-stable-minor.sh

```diff
@@ -34,15 +40,18 @@ fi
 
 if test "$((MINOR % 2))" -eq 0
 then
+	echo "${MAJOR_MINOR} is an EUS release, will create an EUS channel"
 	# 4.even get extended update support: https://access.redhat.com/support/policy/updates/openshift#ocp4_phases
 	CHANNELS="$(printf '%s\n%s' "${CHANNELS}" eus)"
+else
+	echo "${MAJOR_MINOR} is not an EUS release, will not create an EUS channel"
```

Comment:
"will not create an EUS channel" -> "no EUS channel to update"?  This command is `sed`ing to adjust `filters` in channel files that already exist.